index.html

```diff
@@ -193,6 +193,41 @@ <h2>Frequently Asked Questions</h2>
         </div>
     </section>
 
+    <!-- Prizes -->
+    <section id="prizes">
+        <h2>Prizes</h2>
+
+        <img id="prizes-ricky" src="https://via.placeholder.com/980x712.gif" alt="Placeholder image - waiting on gif from design team"></img>
+
+        <div id="prizes-card">
+            <div role="presentation">  <!-- unsure if screen readers will even process <div>s with no text; this may be unnecessary? -->
+                <div class="tape top-left"></div>
+                <div class="tape top-right"></div>
+                <div class="tape bottom-left"></div>
+                <div class="tape bottom-right"></div>
+            </div>
+
+            <div class="content">
+                <img class="graphic" src="https://via.placeholder.com/240x100.png" alt="Placeholder image - waiting on graphic from design team"></img>
+
+                <div class="text">
+                    <p class="title">BRICKHACK MAIL</p>
+                    <p><b>Prizes can only be shipped to United States or Canada residents with active addresses. Remaining eligible attendees can receive swag and/or prizes.</b></p>    
```

Comment:
The last line of this is confusing-- prizes can only be shipped to US and Canada but remaining eligible attendees can receive prizes?

Very dry imo; i don't remember if bh8 had it, but maybe add some sort of more upbeat intro introducing the idea of "we have prizes for different prize tracks"?? Don't have to give specifics just like:

> All BrickHack submissions are eligible to be judged for certain award categories! Prizes and swag can only be shipped to hackers in the United States and Canada with active addresses. Remaining eligible attendees can still receive digital prizes.

Or something like that, I'm not 100% up to date on prizes info